docker/localnet/docker-compose.env

```diff
@@ -0,0 +1,12 @@
+# Environment variables used by Docker Compose
+
+###############################################################
+###             REQUIRED: Node/network selection           ###
+###############################################################
+# Define cheqd-noded image source. Change if using your own build.
+CHEQD_NODE_IMAGE="ghcr.io/cheqd/cheqd-node"
+
+# Define cheqd-noded software release version
+# Current MAINNET recommended version: v0.3.1
+# Current TESTNET recommended version: v0.3.1
+CHEQD_NODE_VERSION="0.3.1"
```

Comment:
> Use this env file to change variables you need within the Docker Compose file.
It's already so.

> Hard-code the published ports for each node.
Already done.

> Then, distinguish the nodes by setting different local IP addresses instead. Docker Compose in bridge mode can be used to define a local network, IP range etc. The advantage is for example if we ever wanted to set up a localnet on AWS, other provides etc where everything isn't just on one machine but across different machines, this can be easily changed by changing the "network" definition. See example in Tendermint repo: https://github.com/tendermint/tendermint/blob/master/docker-compose.yml

Everything is already as you described, even better than in Tendermint.
* Docker-compose creates a bridge network by default
* Instead of custom IPs/network masks we use DNS records that correspond to service names i.e. `observer-0`, 'validator-2', and so on.